spec/01-unit/03-conf_loader_spec.lua

```diff
@@ -67,6 +67,7 @@ describe("Configuration loader", function()
     assert.same(nil, conf.privileged_agent)
     assert.same(true, conf.dedicated_config_processing)
     assert.same(false, conf.allow_debug_header)
+    assert.matches("%d+%.%d+", conf.lmdb_validation_tag)
```

Comment:
Let’s use the version number from meta.lua for the match here. We can make sure the string content is correct too.

Reply:
done.